common.mk

```diff
@@ -458,9 +458,25 @@ define launch-terminal
 		$(LAUNCH_TERMINAL) "$(BUILD_PATH)/soc_term.py $(1)" &
 endef
 else
+tmux := $(TMUX)
+tmux_window := $(shell echo OPTEE_$$RANDOM)
 gnome-terminal := $(shell command -v gnome-terminal 2>/dev/null)
 konsole := $(shell command -v konsole 2>/dev/null)
 xterm := $(shell command -v xterm 2>/dev/null)
+
+ifdef tmux
+define launch-terminal
+	@if tmux list-windows -F '#W' | grep -q $(tmux_window); then \
+		nc -z 127.0.0.1 $(1) || \
+			tmux split-window -d -h -t $(tmux_window) "$(BUILD_PATH)/soc_term.py $(1)" ; \
+	else \
+		nc -z 127.0,0.1 $(1) || \
```

Comment:
typo: s/127.0,0.1/127.0.0.1/

Reply:
Hi, @etienne-lms, when in tmux environment, $TMUX env is automatically set by tmux.
This feature helps developer who is using tmux to open terminal in new tmux window, as the picture below, Two terminals are created in `OPTEE_12293`, but does not support running outside tmux.

![20240724150636_rec_](https://github.com/user-attachments/assets/3cfa355b-de51-4046-8f70-653f72c45468)

Besides, thanks for reviewing and your finding typo mistake, I will fix it right now.

Reply:
Thanks, I better understand now. Works ok (as you would have expected I guess).